Pipfile

```diff
@@ -9,8 +9,7 @@ fastapi = "*"
 uvicorn = "*"
 click = "*"
 boto3 = "*"
-"ga4gh.vrsatile.pydantic" = ">=0.1.0.dev7"
-"ga4gh.vrs" = {version = ">=0.9.0.dev0"}
+"ga4gh.vrs" = {version = "~=2.0.0.dev0"}
```

Comment:
‼️

Reply:
Feels weird, right? We want the VRS and VRS-Python major versions to match